utils/chunked_vector.hh

```diff
@@ -116,6 +116,9 @@ public:
     bool empty() const {
         return !_size;
     }
+    explicit operator bool() const {
+        return !empty();
+    }
     size_t size() const {
```

Comment:
Disagree, chunked_vector is a replacement for std::vector and shouldn't extend its API.

Reply:
Ok.